src/pkgcheck/bash/__init__.py

```diff
@@ -87,13 +91,12 @@ def build_library(output_path, repo_paths):  # pragma: no cover
     from .. import _const
 except ImportError:  # pragma: no cover
     # build library when running from git repo or tarball
-    if not os.path.exists(lib) and 'tree-sitter-bash' in os.listdir(const.REPO_PATH):
+    if not syslib and not os.path.exists(lib) and 'tree-sitter-bash' in os.listdir(const.REPO_PATH):
```

Comment:
As I understood this logic, the `_const.py` module is generated on build/installation (at least on Gentoo), and, as part of that, the vendored tree-sitter-bash is built (as `lang.so`). When running from checkout the _const module doesn't exist and so this ImportError is always thrown (and the vendored lib is built).